doc/releases/pacific.rst

```diff
@@ -2,32 +2,188 @@
 Pacific
 =======
 
-Pacific is the 16th stable release of Ceph.  It is named after the
-giant pacific octopus (Enteroctopus dofleini).
-
-.. DANGER:: DATE: 01 NOV 2021. 
-
-   DO NOT UPGRADE TO CEPH PACIFIC FROM AN OLDER VERSION.  
-
-   A recently-discovered bug (https://tracker.ceph.com/issues/53062) can cause
-   data corruption. This bug occurs during OMAP format conversion for
-   clusters that are updated to Pacific. New clusters are not affected by this
-   bug.
-
-   The trigger for this bug is BlueStore's repair/quick-fix functionality. This
-   bug can be triggered in two known ways: 
-
-    (1) manually via the ceph-bluestore-tool, or 
-    (2) automatically, by OSD if ``bluestore_fsck_quick_fix_on_mount`` is set 
-        to true.
+v16.2.7 Pacific   
+===============
 
-   The fix for this bug is expected to be available in Ceph v16.2.7.
+This is the seventh backport release in the Pacific series.
 
-   DO NOT set ``bluestore_quick_fix_on_mount`` to true. If it is currently
-   set to true in your configuration, immediately set it to false.
+Notable Changes
+---------------
 
-   DO NOT run ``ceph-bluestore-tool``'s repair/quick-fix commands.
+* Critical bug in OMAP format upgrade is fixed. This could cause data corruption
+  (improperly formatted OMAP keys) after pre-Pacific cluster upgrade if
+  bluestore-quick-fix-on-mount parameter is set to true or ceph-bluestore-tool's
+  quick-fix/repair commands are invoked.
+  Relevant tracker: https://tracker.ceph.com/issues/53062
+  ``bluestore-quick-fix-on-mount`` continues to be set to false, by default.
+  
+  If you are not using cephadm, you must disable FSMap sanity checks *before starting the upgrade*::
```

Comment:
Something wrong with the formatting here. Shouldn't it be a separate bullet and formatted like the other entries?
Verbatim I propose:
```
* CephFS: If you are not using cephadm, you must disable FSMap sanity checks *before starting the upgrade*::

      ceph config set mon mon_mds_skip_sanity true

  After the upgrade has finished and the cluster is stable, please remove that setting::

      ceph config rm mon mon_mds_skip_sanity

  Clusters managed by and upgraded using cephadm take care of this step automatically.
```

Reply:
Oh, I thought it was an addendum to the first bullet point.  Will fix.

Reply:
> Oh, I thought it was an addendum to the first bullet point. Will fix.

doh, sorry, i realize now my comment earlier was misleading. Indeed it's a separate point. Sorry about that!